lib/modules/manager/nuget/__fixtures__/with-container-base-image/with-pinned-digest.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk.Worker">
```

Comment:
inline small fixtures as requested

Reply:
Are you asking me to copy pase the content of the two fixtures and paste it in the extract.spec.ts file?

Reply:
yes, like already done for other tests

Reply:
Is it ok for you ?